### PR TITLE
MAINT: updates `diversity.methods.alpha` to `pipelines.alpha`

### PR DESCRIPTION
**Pull request by ChrisKeefe**
~Also, miscellaneous copy edits in PD Mice tutorial~
PD Mice tutorial edits have been moved to #488

### CODE REVIEW

**Comment by ChrisKeefe** [comment]
[forum xref](https://forum.qiime2.org/t/im-having-a-technical-issue-following-the-artifact-api-page/16614/4) for diversity.methods->pipelines

**Comment by thermokarst** [comment]
Please move copy edits to #488

**Comment by thermokarst** [comment]
@ChrisKeefe - please move copy edits to #488.

**Comment by thermokarst** [comment]
Mark this a "ready for review" when you're all set, @ChrisKeefe.

**Comment by ChrisKeefe** [comment]
@thermokarst, I've added a couple of `artifact-api` copy edits to this PR. Do you want those here, in #488, or in a separate PR?

Once that question's handled, this will be ready for review.

**Comment by thermokarst** [comment]
Here makes sense to me.

To be clear, my request for you to remove copy edits wasn't a blanket statement for _all_ copy edits, just for the PD Mice ones, since they A) had nothing to do with this PR, B) we already had a plan in place to handle the bulk updates for copy edits to the PD Mice tutorial, and C) collaborating on a git branch is a useful exercise for the lab.

**Comment by thermokarst** [comment]
When you're ready for review, please:

1. Mark as "ready for review" 
2. Move this card into the "in review" column on the project board
3. Unassign yourself

Thanks!

File: source/interfaces/artifact-api.rst [hunk 1]
```diff
@@ -76,7 +76,7 @@ Finally, we can save our ``Artifacts`` as ``.qza`` files and exit the interprete
 
    >>> rarefied_table.save('rare.qza')
    'rare.qza'
-   >>> alpha_diversity.save('oo.qza')
+   >>> alpha_diversity.save('obs_features.qza')
    'oo.qza'
```
Comment (andrewsanchez) [inline source/interfaces/artifact-api.rst:80]:
```suggestion
   'obs_features.qza'
```

**Comment by andrewsanchez** [comment]
There was one minor correction I made to an output file name but other than that, LGTM! 🌮

**Pull request merged**